projectile.el

```diff
@@ -1146,7 +1146,7 @@ Controlled by `projectile-require-project-root'."
     (cond
      ((eq projectile-require-project-root 'prompt) (projectile-completing-read
                                                     "Switch to project: " projectile-known-projects))
-     (projectile-require-project-root (error "Projectile can't find a project definition in %s" dir))
+     (projectile-require-project-root (error "Projectile is unable to find a project definition in %s" default-directory))
```

Comment:
@bbatsov I feel really bad for having changed the error message just to make the tests pass. The reason the test was failing was because of a mismatching quote:

`can't` versus `can’t`

Note how the quote character is different. This failure only happens in CI (and not on my local) and I'm not sure why. The character both from the error message and the test is a normal single quote but for some reason the CI environment is replacing it with a `’`. Any pointers about this let me know. In the meantime, I will try to figure it out.